### ISSUE TITLE
Bad Accuracy

### ISSUE THREAD

**Issue by shivanidabas31** [chiranthancv95/Handwritten-Line-Text-Recognition-with-Tensorflow#1]
Hi Chiranthan,

This program is providing an accuracy of 4% and CER 16.8% after training for 24hours. 
Please look into it. Even the simplest words can't be recognised.

**Comment by chiranthancv95** [comment]
Yeah, I'll look into it. I got busy with some other things. I'll try to improve it once everything gets sorted out.
Thank you for your interest.
